packages/components/src/FormField/FormField.css

```diff
@@ -234,7 +234,7 @@
   padding-top: var(--field--padding-top);
   /* Remove padding on right to make use of the entire width */
   padding-right: 0;
-  transform: translateY(--field--placeholder-transform);
+  transform: translateY(var(--field--placeholder-transform));
```

Comment:
The `stylelint-config-standard` upgrade here 7f0a4aa75131723854547178a6816a1bce6c6694 let me know we were using this token without a `var` 👍

Reply:
From the MDN `var` should be used to reference custom properties (see below). This usage should have been a CSS var so I am not concerned with this change and things seem do to still work on the formfield

[Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)